src/operator/mshadow_op.h

```diff
@@ -12,6 +12,13 @@
 namespace mxnet {
 namespace op {
 namespace mshadow_op {
+#ifdef __CUDA_ARCH__
+#define CONSTANT __constant__
+#else
+#define CONSTANT
```

Comment:
no constant can't build on cuda. constexpr vs2013 not support.
i try change it to
#ifdef __CUDA_ARCH__
__constant__ const float PI = 3.14159265358979323846;
#else
const float PI = 3.14159265358979323846;
#endif